flagcx/core/group.h

```diff
@@ -79,69 +72,17 @@ extern __thread struct flagcxIntruQueue<struct flagcxAsyncJob,
                                         &flagcxAsyncJob::next>
     flagcxAsyncJobs;
 
-static flagcxResult_t groupLaunch(struct flagcxAsyncJob *job_);
+flagcxResult_t flagcxGroupErrCheck(flagcxResult_t ret);
+void flagcxGroupCommJoin(struct flagcxHeteroComm *comm);
```

Comment:
Consider using typedef to define your structs instead of always using struct explicitly. This improves code readability and reduces verbosity when declaring variables.